### PR TITLE
Update unit test coverage

### PR DESCRIPTION
**Pull request by alex-deaconu**
## Description
Update unit test coverage

Move integration related store api calls
to billing for consistency

[stage-15]

## Motivation and Context
Refactoring and test coverage.

## How Has This Been Tested?
Tested changes locally and updated unit tests.

## Release Plan:
- As the Submitter, upon requesting review of this pull request, I confirm that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed. 
- As the Reviewer, upon approving the changes in this PR, I confirm I have reviewed and I agree that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed

#### Release Checklist Items Skipped?
No

### CODE REVIEW

**Comment by alex-deaconu** [comment]
@ezequielc please review. I will need to also fix the failing e2e test for this, somehow the spinner is still showing when we check for the button presence. Thanks.

File: test/unit/purchase/app.tests.js [hunk 1]
```diff
@@ -196,28 +232,6 @@ describe('app:', function() {
         done();
       },10);
     });
-
-    it('should resolve redirectTo as previous path', function() {
```
Comment (ezequielc) [inline test/unit/purchase/app.tests.js:200]: @alex-deaconu Just to confirm, the `redirectTo` is now part of the abstract route, so this test no longer works? Just want to make sure I understand this. Thanks

File: test/unit/purchase/controllers/ctr-update-subscription-spec.js [hunk 2]
```diff
@@ -238,6 +242,20 @@ describe("controller: add-licenses", function() {
       $scope.close();
 
       $location.path.should.have.been.calledWith(redirectTo);
+      $state.go.should.not.have.been.called;
+    });
+
+    it("should close modal and redirect to provided path", function() {
```
Comment (ezequielc) [inline test/unit/purchase/controllers/ctr-update-subscription-spec.js:248]: @alex-deaconu I think the description is incorrect here. I assume it should redirect to the subscription page if no redirect is provided. Thanks

**Review by ezequielc — APPROVED** [review]
@alex-deaconu Looks good, minor comments inline. Thanks 👍

File: test/unit/purchase/controllers/ctr-update-subscription-spec.js (hunk 2 above)
Comment (alex-deaconu) [reply test/unit/purchase/controllers/ctr-update-subscription-spec.js:248]: @ezequielc yeah I noticed these descriptions, they will be updated in my next PR. Thanks.

File: test/unit/purchase/app.tests.js (hunk 1 above)
Comment (alex-deaconu) [reply test/unit/purchase/app.tests.js:200]: @ezequielc the tests were just moved up to the abstract route, but still reference the add/remove routes. Thanks.

Comment (ezequielc) [reply test/unit/purchase/app.tests.js:200]: @alex-deaconu Sounds good. Thanks!

**Pull request merged**